pkg/client/cmd/install.go

```diff
@@ -57,15 +59,21 @@ func (o *InstallCmdOptions) install(cmd *cobra.Command, args []string) error {
 		return nil // TODO better error handling: if here we return err the help page is shown
 	}
 
-	if o.ClusterSetupOnly {
+	if o.clusterSetupOnly {
 		fmt.Println("Camel K cluster setup completed successfully")
 	} else {
 		namespace := o.Namespace
 
-		err = install.InstallOperator(namespace)
+		err = install.Operator(namespace)
 		if err != nil {
 			return err
 		}
+
+		err = install.PlatformContexts(namespace)
```

Comment:
It's ok for now. Later with #73 it will be the operator to install those resources. Installation should be limited to the minimum set of dependencies (operator, crd and rbac).